### ISSUE TITLE
Include keywords in badge detection

### ISSUE THREAD

**Issue by Hirse** [dnbard/brackets-extension-rating#117]
As far as I can see only the title (and for some of them the description) is used for the badges.
Could you include the keywords in that?

**Comment by dnbard** [comment]
Yes, I think so. Need to check that, but I think that it is possible.
